Automattic/sensei
# Fix responsive styles and tweak prev/next links

Fixes responsive design issues.
Fixes #4449

### Changes proposed in this Pull Request

* It fixes the responsive design issues in the learning mode.
* It updates the prev/next button to be displayed as disabled when it's in the first/last lesson.
* It moves the prev/next classes to a single class.
* It improves accessibility by hiding the footer too when collapsing sidebar.
* It updates the "Expand" button to be full width with a background to avoid being over the content.

### Testing instructions


* Create a course with many lessons.
* Enable the learning mode for this course.
* Go to the learning mode as a student and as an admin, resize the window to different sizes, open and close the menu, and make sure everything works properly.
* Also make sure you can navigate properly through the Prev/Next buttons on the top bar.
* Navigate to the first lesson, and make sure you see the Previous link as disabled.
* Navigate to the last lesson, and make sure you see the Next link as disabled.


### Screenshot / Video

Video showing the responsive design (The part with the inspector open was only to see it working wider than `600px` and smaller than `600px`):

https://user-images.githubusercontent.com/876340/151233011-3ba489f1-9935-449b-b93e-cc52bdb3b505.mp4

<img width="354" alt="Screen Shot 2022-01-25 at 16 47 09" src="https://user-images.githubusercontent.com/876340/151048730-5e7a6801-4334-4165-bca2-d1dd6e06dcdc.png">
